pkg/apis/extensions/v1beta1/doc.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/extensions
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/autoscaling
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/policy
```

Comment:
see if you can put this line first... I'd expect it to look for matching types in the internal policy package first